manager/controllers/advisories.go

```diff
@@ -39,7 +39,7 @@ type AdvisoriesDBLookup struct {
 // nolint: lll
 type AdvisoryItemAttributes struct {
 	SystemAdvisoryItemAttributes
-	ApplicableSystems int `json:"applicable_systems" query:"COALESCE(aad.systems_affected, 0)" csv:"applicable_systems" gorm:"column:applicable_systems"`
+	ApplicableSystems int `json:"applicable_systems" query:"COALESCE(aad.systems_installable, 0)" csv:"applicable_systems" gorm:"column:applicable_systems"`
```

Comment:
shouldn't we show `aad.systems_installable + aad.systems_applicable` on advisories list page? According to mocks, it should show Affected systems which should be sum of installable and applicable systems